specification/sql/resource-manager/readme.md

```diff
@@ -150,7 +149,7 @@ input-file:
 - ./Microsoft.Sql/preview/2021-02-01-preview/DatabaseUsages.json
 - ./Microsoft.Sql/preview/2021-02-01-preview/LedgerDigestUploads.json
 - ./Microsoft.Sql/preview/2021-02-01-preview/OutboundFirewallRules.json
-- ./Microsoft.Sql/preview/2021-02-01-preview/ReplicationLinks.json
+- ./Microsoft.Sql/preview/2021-05-01-preview/ReplicationLinks.json
```

Comment:
can we use the latest API version: 2021-11-01-preview? 
can we also follow the alphabetical order to sort this section?

Reply:
but the most up-to-dated preview version I found under "D:\azure-rest-api-specs\specification\sql\resource-manager\Microsoft.Sql\preview" is 2021-05-01-preview.
![image](https://user-images.githubusercontent.com/92422693/166569039-ece95e1d-643c-4710-9ed6-41cd436c847f.png)

Reply:
feel free to git pull origin main
https://github.com/Azure/azure-rest-api-specs/tree/main/specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview